blazor/dropdown-list/localization.md

```diff
@@ -7,6 +7,23 @@ control: DropDown List
 documentation: ug
 ---
 
-# Localization in Blazor DropDown List Component
+# Globalization and Localization in Dropdown List
+
+## Localization
+
+[Blazor DropDownList](https://www.syncfusion.com/blazor-components/blazor-dropdown-list) component can be localized. Refer to [Blazor Localization](https://blazor.syncfusion.com/documentation/common/localization) topic to localize Syncfusion Blazor components.
```

Comment:
The [Blazor DropDownList](https://www.syncfusion.com/blazor-components/blazor-dropdown-list) component can be localized. Refer to the [Blazor Localization](https://blazor.syncfusion.com/documentation/common/localization) topic to localize Syncfusion Blazor components.